mypy/server/aststrip.py

```diff
@@ -196,6 +199,14 @@ def visit_import(self, node: Import) -> None:
                     symnode.kind = UNBOUND_IMPORTED
                     symnode.node = None
 
+    def visit_import_all(self, node: ImportAll) -> None:
```

Comment:
You might need an `is_unreachable` check here to avoid running into trouble with imports in `if MYPY` else blocks. (I needed to add such a check to `visit_import`  and `visit_import_from`.)

Reply:
Good point, will add.